src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -905,3 +904,30 @@ def perturb_individual_age(self, pop: pd.DataFrame) -> pd.Series:
         # Clip ages above 99 at 99
         pop.loc[pop["age"] > 99, "age"] = 99
         return pop["age"]
+
+    def assign_last_name_ids(self, pop: pd.DataFrame) -> pd.Series:
+        # Assigns index value as last name id
+        # Matches last name id for relatives of a reference person within a household
+        # Note: Group quarters are ignored for matching last name ids
+
+        pop["last_name_id"] = pop.index
+
+        # Match last names for relatives of reference person
+        relatives_idx = pop.loc[
```

Comment:
Looks good, but I think I am in favor of using `pop.index[...]` over `pop.loc[...].index`